src/Services/ArrayManager.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Helpers\Services;
+
+class ArrayManager
+{
+    /**
+     * Returns TRUE if var is an array, FALSE otherwise.
+     * @param array $value
+     * @return bool
+     */
+    public function isArray(array $value): bool
+    {
+        if (count($value) === 0) {
+            return false;
+        }
+        return is_array($value[0]);
```

Comment:
Why are you checking only first index of an array. I think you should check entire array right? **is_array($value);**